### PR TITLE
LPS-186052 - Fix Workflow | Automation | Master

### PR DESCRIPTION
**Pull request by thaiscabrals**
**Ticket:** [LPS-186052](https://issues.liferay.com/browse/LPS-186052)
**Tests:** 
- KaleoFormsNewProcess#NewProcessAppearsOnProcessBuilder
- KaleoFormsNewProcess#CanOnlyChooseOneWorkflow

### CODE REVIEW

**Comment by thaiscabrals** [comment]
ci:test:sf

**Comment by thaiscabrals** [comment]
ci:test:sf

**Comment by thaiscabrals** [comment]
ci:test

**Comment by patriciajeanperez** [comment]
Hey @thaiscabrals, looks like the test just made the window bigger but the scroll is still there. Looks like a UI bug to me but can you compare this behavior in an older bundle when the tests were still passing to see how it looked like? Thank you!

File: modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/macros/KaleoFormsAdmin.macro [hunk 1]
```diff
@@ -525,10 +525,7 @@ definition {
 	}
 
 	macro publishWorkflow {
-		AssertClick(
-			key_text = "Publish",
-			locator1 = "Button#ANY",
-			value1 = "Publish");
+		Click(locator1 = "Button#PUBLISH");
```
Comment (thaiscabrals) [inline modules/dxp/apps/portal-workflow/portal-workflow-kaleo-forms-test/src/testFunctional/macros/KaleoFormsAdmin.macro:528]: The Publish button has changed.

**Comment by thaiscabrals** [comment]
ci:test:workflow

**Comment by thaiscabrals** [comment]
ci:test:workflow

**Comment by thaiscabrals** [comment]
ci:test:workflow

**Comment by patriciajeanperez** [comment]
@thaiscabrals can we check the test results and confirm if the affected tests are passing? Thanks!

**Comment by patriciajeanperez** [comment]
Hey @thaiscabrals, I only saw one test for that ticket. How about the other tests affected by that macro? In other words, were the test results above passing with no unique failures?

**Comment by thaiscabrals** [comment]
Hey @patriciajeanperez, These are not unique failures. All tests using the changed macros had been tested locally. I have added a comment with the results to the ticket. 
Thank you.

**Comment by patriciajeanperez** [comment]
ci:forward

**Pull request closed without merging**